.buildkite/pipeline.yml

```diff
@@ -54,10 +54,10 @@ steps:
       ROOT_VOLUME_SIZE: 500
       SCALE_UP_ADJUSTMENT: 1
       AGENTS_PER_INSTANCE: 5
-      SPOT_PRICE: 0.2670
+      SPOT_PRICE: 0.2660
```

Comment:
I like that we let the spot instance market dertermine if we run our builds. 🤣

Reply:
Are you not a gambling man?